src/resources/js/pages/Users.tsx

```diff
@@ -1,5 +1,66 @@
-import { strings } from '../localization';
+import Box from '@mui/material/Box';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { User } from 'bmlt-root-server-client';
+import { useEffect, useState } from 'react';
+
+import RootServerApi from '../RootServerApi';
 
 export const Users = () => {
-  return <div>{strings.usersTitle}</div>;
+  const [currentSelection, setCurrentSelection] = useState('create-new-user');
+  const [selectedUser, setSelectedUser] = useState<any>({});
+  const [users, setUsers] = useState<User[]>([]);
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setCurrentSelection(event.target.value as string);
+    users.forEach((user) => {
+      if (event.target.value === user.username) {
+        setSelectedUser(user);
+      } else if (event.target.value === 'create-new-user') {
+        setSelectedUser({});
+      }
+    });
+  };
+
+  const getUsers = async () => {
+    try {
+      const allUsers = await RootServerApi.getUsers();
+      setUsers(allUsers);
+      console.log(allUsers);
+    } catch (error: any) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    getUsers();
+  }, []);
+
+  return (
+    <div>
+      <Box>
+        <FormControl>
+          <InputLabel id='select-label'>User</InputLabel>
```

Comment:
Everwhere you add a string, add an english translation to `localization.ts`. Then you can import `strings` and use the translation.